LondonBritishLibrary/orient/BLorient757.xml

```diff
@@ -31,23 +31,126 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 335</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
                         <summary/>
-                        
-                        
-                        
-                        
+                        <msItem xml:id="ms_i1">
+                          <locus from="4r"></locus>
+                          <title type="complete" ref="LIT1948Mashaf"></title>
+                          <textLang mainLang="gez"></textLang>
+                          <note>With many marginal notes, some of them in Amharic.</note>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                          <locus from="131r"></locus>
+                          <title type="complete" ref="LIT2050Nagara"></title>
+                          <textLang mainLang="gez"></textLang>
+
+                          <msItem xml:id="ms_i2.1">
+                            <locus from="131r"></locus>
+                            <title type="complete" ref="LIT2050Nagara#IsaiahOfScete" xml:lang="en">
+                              Extract from Isaiah of Scete
```

Comment:
Maybe "excerpt" is better? Or nothing has to be written in the title, if the work record is elaborated.